tests/test_subsets.py

```diff
@@ -6,39 +6,39 @@
 
```

Comment:
Instead of writing out separate test functions for each test case, you should really be using parametrization. This uses less lines of code, prevents repetition, and improves maintainability.